src/bit_str.jl

```diff
@@ -231,7 +231,7 @@ onehot(n::BitStr, nbatch::Int) = onehot(Float64, n, nbatch)
 Return left-right reflected bit string.
 """
 breflect(b::BitStr{N}) where N = breflect(b; nbits=N)
-breflect(b::BitStr{N}, masks::Vector{<:Integer}) where N = BitStr{N}(breflect(Integer(b), masks; nbits=N))
+breflect(b::BitStr{N,T}, masks::Vector{<:BitStr{N,T}}) where {N,T} = BitStr{N}(breflect(Integer(b), reinterpret(T,masks); nbits=N))
```

Comment:
I think it is OK to allow to use integers as mask?

Reply:
I made this restriction to avoid the ambiguity error.

Reply:
what's the ambiguity error?

Reply:
`@inline function breflect(b::T, masks::AbstractVector{T}; nbits::Int)::T where T<:Integer`